core/translation.cpp

```diff
@@ -1171,13 +1171,11 @@ void TranslationServer::_bind_methods() {
 void TranslationServer::load_translations() {
 
 	String locale = get_locale();
-	bool found = _load_translations("locale/translations"); //all
+	_load_translations("locale/translations"); //all
```

Comment:
OK for this change, the `found` logic was used for a now-removed fallback logic: 0b806ee0fc